readme.md

```diff
@@ -1,10 +1,29 @@
 # ovos-PHAL-plugin - alsa volume control
 
-controls system volume with alsa
+## Install
+
+```sh
+python -m pip install git+https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/OpenVoiceOS/ovos-PHAL-plugin-alsa
+```
+
+## Configure
+
+In your mycroft.conf
+
+```json
+"PHAL": {
+    "ovos-PHAL-plugin-alsa": {"enabled": true}
```

Comment:
this is not needed, it is the default config, "enabled": False would stop the plugin from loading if you want to document that instead

Reply:
then it should be 
```json
"PHAL": {
    "ovos-PHAL-plugin-alsa": {}
}
```
?

Reply:
no config step is needed